.github/workflows/changelog.yml

```diff
@@ -12,3 +12,7 @@ jobs:
           changelog_regex: "CHANGELOG.md"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Changelog Tool
+        uses: rcmachado/changelog-action@v1
+        with:
+          args: init --output CHANGELOG.md --compare-url https://keepachangelog.com/en/1.0.0/
```

Comment:
@camilamaia is this the correct way of using the Changelog Tool or should I keep this in the `lint.yml`

Reply:
@Pradhvan I believe the command can be the same we are already using: https://github.com/scanapi/scanapi/blob/master/.circleci/config.yml#L17

From what I understood, init would create a new file, but here we only want to check the lint, right?

Reply:
@camilamaia You're correct - `init` will initialize a new file. But as there is no `lint` command yet, the `diff` command should be used instead (there is an issue open to implement it: https://github.com/rcmachado/changelog/issues/71).

Reply:
I tried the diff[0] but it's not working. I you can take a look @rcmachado it would be a great help. 

[0]https://github.com/scanapi/scanapi/pull/246/checks?check_run_id=932150102